pages/donate.md

```diff
@@ -119,11 +121,15 @@ However, if you or your organization has relevant knowledge, experience, leads,
 
 The donation address can be verified via the [Qubes Security Pack][Qubes Security Pack], specifically in the [fund directory][fund].
 We provide [detailed instructions for verifying the digital signatures][verify].
-You can also view the address on [blockchain.info].
```

Comment:
Can we keep this link somewhere (maybe indeed verification section is not appropriate)? I personally use it quite often. Maybe besides the address itself?

Reply:
I've heard complaints about the choice of blockchain explorer here. Apparently some people really don't like blockchain.info (which now redirects to .com) for some reason, and I see no reason for us to appear to be "taking sides" when there is really no reason for us to link to a blockchain explorer in the first place. (Anyone who wants to see the address in a blockchain explorer can simply navigate to their explorer of choice and copy/paste the address, then bookmark it.)

I fear that "elevating" the link to the top section and making it even more visible will only exacerbate this problem.

I agree that the link has nothing to do with verification, and I don't want us to appear to be under the misconception that linking to an address on a blockchain explorer somehow serves to authenticate that address, nor do I wish to mislead others into believing this.

TBH, I was hesitant to include this change in this commit, since it's not really related to the major change here. I figured no one would care, so I went ahead, but that was sloppy of me, and I apologize.

Reply:
> I agree that the link has nothing to do with verification, and I don't want us to appear to be under the misconception that linking to an address on a blockchain explorer somehow serves to authenticate that address, nor do I wish to mislead others into believing this.

Makes sense.

Reply:
> TBH, I was hesitant to include this change in this commit, since it's not really related to the major change here. I figured no one would care, so I went ahead, but that was sloppy of me, and I apologize.

I'm going to back this change out of this PR and handle it in a separate one.